gammapy/time/plot.py

```diff
@@ -30,3 +28,140 @@ def plot_time_difference_distribution(time, ax=None):
 
     # TODO: implement!
     raise NotImplementedError
+
+def plot_light_curve(name_3FGL, time_start, time_end, ax = None):
+    """Plot flux as a function of time for a fermi 3FGL object.
+
+    Parameters
+    ----------
+    name_3FGL : `string`
+        The 3FGL catalog name of the object to plot
+    fermi_met_start : `~astropy.time.Time`
+        Astropy time object for the start of the light curve
+    fermi_met_end : `~astropy.time.Time`
+        Astropy time object for the end of the light curve
+    ax : `~matplotlib.axes.Axes` or None
+        Axes
```

Comment:
The formatting (the underlining) of the `Returns` section doesn't look OK:
![screen shot 2015-06-14 at 21 00 28](https://cloud.githubusercontent.com/assets/852409/8149999/7ebe2512-12d8-11e5-91a7-a557b2c6b9af.png)

I think the issue is that you have to add an empty line.
It also might be that you have to give a one-line description of the `ax` return object.

You can use these commands to check locally:
```
python setup.py build_sphinx
open docs/_build/html/index.html
```
Yes, the Sphinx build is annoyingly slow. I don't know how to make it faster.